packages/common/core-interfaces/src/messages.ts

```diff
@@ -13,7 +35,7 @@
  *
  * See at `server/routerlicious/packages/lambdas/src/utils/messageGenerator.ts`.
  */
-export interface ISignalEnvelope {
+export interface ISignalEnvelope<TMessage extends TypedMessage = TypedMessage> {
 	/**
 	 * The target for the envelope, undefined for the container
```

Comment:
nit:
```suggestion
	 * The target for the envelope, expected to be undefined since these target the container itself
```

Reply:
I guess that's not right, I see it used in ContainerRuntime.  I was misinterpreting the doc comment for the interface, which seems to indicate that address will always be missing.